utils/__mocks__/ifPortIsFreeDo.js

```diff
@@ -0,0 +1,3 @@
+const mock = jest.fn((port, todo) => { todo(); });
```

Comment:
This may be silly but can we put __mocks__ inside their __tests__ folders? 
And then any that would be used in the top level could be in __mocks__ at the root?